tests/runtest.proj

```diff
@@ -379,6 +379,26 @@ namespace $([System.String]::Copy($(Category)).Replace(".","_").Replace("\","").
     <MSBuild Projects="$(MSBuildProjectFile)" Targets="UploadPerfAssemblies" />  
   </Target>  
 
+  <PropertyGroup>
+    <CMakeCache>$(RootBinDir)obj\$(BuildOS).$(BuildArch).$(BuildType)\CMakeCache.txt</CMakeCache>
+    <CMakeCacheContent>$([System.IO.File]::ReadAllText($(CMakeCache)))</CMakeCacheContent>
```

Comment:
Does this load the entire cache into memory? How big is the cache? I'm wondering whether we need to worry about how resource-intensive this is, or if it's safe as-is.